index.js

```diff
@@ -134,7 +134,7 @@ module.exports = class RAM extends RandomAccess {
   }
 
   _unlink (req) {
-    this._buffers = []
+    this.buffers = []
     this.length = 0
```

Comment:
I was wondering for a bit if `pageSize` should also be reset here but I guess it makes sense for it to keep its initial value from the constructor.